scheduler/common/cluster-client/src/main/kotlin/ac/uk/ebi/cluster/client/lsf/ClusterOperations.kt

```diff
@@ -7,18 +7,24 @@ import ac.uk.ebi.cluster.client.model.JobSpec
 import arrow.core.Try
 import com.jcraft.jsch.JSch
 import com.jcraft.jsch.Session
+import ebi.ac.uk.coroutines.waitUntil
 import mu.KotlinLogging
+import java.time.Duration
 
+private const val CHECK_COMMAND = "bjobs -o STAT -noheader %s"
+private const val DONE_STATUS = "DONE"
 private const val REDIRECT_LOGS = "-o $LOGS_PATH%J_OUT -e $LOGS_PATH/%J_IN"
 private const val SUBMIT_COMMAND = "bsub $REDIRECT_LOGS"
+
 private val logger = KotlinLogging.logger {}
 
+// TODO I think this could be moved to the :commons project since we'll be moving away from the scheduler as an app
```

Comment:
I think project can be renamed to be clusterClient and being under clients project

Reply:
While this is being refactored, it might make sense to externalise CHECK/SUBMIT commands and remove `lsf` from the package name since the same code might be resused for slurm